test/build_and_run_fuzz_targets.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if [[ $# -gt 0 ]]; then
+  FUZZ_TARGETS=$*
+else
+  echo "This script should be called from tools/run_envoy_bazel_coverage.sh"
+fi
+
+LIBFUZZER_TARGETS=""
+# Build all fuzz targets to run instrumented with libfuzzer in sequence.
+for t in ${FUZZ_TARGETS}
+do
+  LIBFUZZER_TARGETS+="${t}_with_libfuzzer "
+done
+
+bazel build ${LIBFUZZER_TARGETS} --config asan-fuzzer -c opt
```

Comment:
Should we be using ${BAZEL_BUiLD_OPTIONS} here @lizan ?

Reply:
Yes unless there's reason not to do so.

Reply:
ping?

Reply:
Done! Sorry about that